parser_library/src/workspace_manager.cpp

```diff
@@ -319,14 +319,19 @@ class workspace_manager_impl final : public workspace_manager,
 
         std::chrono::duration<double> duration = std::chrono::steady_clock::now() - start;
 
-        const auto& [url, metadata, perf_metrics, errors, warnings] = task.value();
+        const auto& [url, metadata, perf_metrics, errors, warnings, outputs_changed] = task.value();
```

Comment:
It's almost asking for a struct as a return value now 😄

Reply:
There is one, it's just been practical to immediately use structured binding.